DigitalLearningSolutions.Data/Services/DelegateApprovalsService.cs

```diff
@@ -25,19 +26,24 @@ public class DelegateApprovalsService : IDelegateApprovalsService
         private readonly IEmailService emailService;
         private readonly ILogger<DelegateApprovalsService> logger;
         private readonly IUserDataService userDataService;
+        private readonly IConfiguration config;
+
+        private string? LoginUrl => config["AppRootPath"] + "/Login";

```

Comment:
Is this actually nullable?

Reply:
I thought the IDE was of the opinion it was, but now it seems otherwise. Maybe a leftover from some earlier version.